pkg/gui/style/basic_styles.go

```diff
@@ -27,6 +27,21 @@ var (
 
 	AttrUnderline = New().SetUnderline()
 	AttrBold      = New().SetBold()
+
+	ColorMap = map[string]struct {
```

Comment:
I hope this is OK; exporting ColorMap to use everywhere else. Want to automatically expose all colors to the template.

Reply:
At first i was like let's not do this but seeing the duplicated code in  `pkg/theme/style.go` now removed lets me think this is better.
Lets keep your change